java/ql/lib/semmle/code/java/security/RsaWithoutOaepQuery.qll

```diff
@@ -0,0 +1,15 @@
+/** Definitions for the RSE without OAEP query */
+
+import java
+import Encryption
+import semmle.code.java.dataflow.DataFlow
+
+/** Holds if `c` is a call which initialises an RSA cipher without using OAEP padding. */
+predicate rsaWithoutOaepCall(CryptoAlgoSpec c) {
+  exists(CompileTimeConstantExpr specExpr, string spec |
+    specExpr.getStringValue() = spec and
+    DataFlow::localExprFlow(specExpr, c.getAlgoSpec()) and
```

Comment:
Do you think it would make sense to use a full-blown `DataFlow::Configuration` here so that we catch some more cases (e.g. field flow or inter-procedural flow)?